test/forms/labels/labeled-field.test.js

```diff
@@ -10,9 +10,9 @@ test('wraps children in fieldset', () => {
       <Wrapped />
```

Comment:
I believe the test description needs to be rewritten to not directly reference `fieldset`

Reply:
Could we add a test that explicitly tests the `as="fieldset"` case? E.g.,

`expect(screen.getByRole('group')).toBeInTheDocument()`

Reply:
@chawes13 I'm actually not sure what this first test is accomplishing... I'm merely looking for an element with the test-id I assigned and testing that it exists (without 'error' class). Should we keep it?

Reply:
Yes we should keep it. We have several components **and** styling that rely on LabeledField returning a containing element (div or fieldset), as opposed to a fragment

Reply:
Now that you're adding `'field-wrapper'` to this container, that'd be a useful thing to assert on (especially since our styles depend on it!)